server/bundler/utils.js

```diff
@@ -14,6 +14,9 @@ function getAssets( stats ) {
 }
 
 function pathToRegExp( path ) {
+	if ( path === '/' ) {
+		return path;
+	}
 	return new RegExp( '^' + path + '(/.*)?$' );
```

Comment:
for what it's worth, **lodash** provides [escapeRegExp](https://lodash.com/docs/4.17.4#escapeRegExp) which would prevent such errors.

maybe a more important question to ask, especially since we're responding to invalid URLs is why we aren't using a full-out URL parser?

what's our goal? what should this code going to accomplish? are we just trying to produce a thing that will find URLs matching a certain path and then gathering up the remaining part of the URL?

do we have any constraints on path? must it be `a/slash/separated/string`?

what is the value of the `/` within the group? mixtures of `*` and `?` don't usually sit well.

❓ ❓ just more questions